_posts/2018-07-21-IWD.md

```diff
@@ -0,0 +1,51 @@
+# International Women's Day (IWD)
+
+A fabulous event organized by **WTM(Women Techmakers) held on 8th April 2018** was all about how far women have reached, where they are falling behind and how perceptions continue to shift. It was one of the memorable meets of [GDG Baroda](https://www.meetup.com/gdgbaroda/) to empower women.
+
+![IWD](https://scontent.fbdq1-1.fna.fbcdn.net/v/t1.0-9/30572137_951380235031037_4460753643510956032_n.jpg?_nc_cat=0&oh=0e09ffe70e44653aaddfc710f3f796ca&oe=5BCCDC63)
```

Comment:
Please download the images in the assets folder and use from there. These links can expire anytime.